### PR TITLE
Provided a new version of the structure on README, the goal is to allow users able to install and use the software without your support.

### PR DESCRIPTION
**Pull request by mfan7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+## Installation Instructions
+
+`TODO: rewrite the following instructions with specific details, for example, which version of Windows, which specific version of JDK needed, what are the [relevant libraries] and [encoding, decoding] tools, how to compile and run, etc.`
 
 1. The software is currently only tested on Windows systems and may not be compatible with other systems
 2. The computer is equipped with JDK
 3. Install relevant libraries and encoding and decoding tools
 4. You can compile and run on the terminal or import projects in the IDE
 
+### Dependance Installation
```
Comment (XavierBohuaLiu) [inline README.md]: Could you fix the typo `Dependance`

**Review by XavierBohuaLiu — COMMENTED** [review]
Submit the comment

File: README.md (hunk 1 above)
Comment (mfan7) [reply README.md]: https://github.com/XavierBohuaLiu/StudentManagement/pull/1/commits/1daf564ca61b2ee8563f31cf154397e3ce892be7

**Review by XavierBohuaLiu — APPROVED** [review]
Looks good to me.

**Pull request merged**